udata/core/organization/api_fields.py

```diff
@@ -65,10 +65,15 @@ def check_can_access_email():
     user_ref_fields,
     {
         "email": fields.Raw(
-            attribute=lambda o: o.email if check_can_access_email() else None,
+            attribute=lambda o: o.email if check_can_access_user_private_info() else None,
             description="The user email (only present on show organization endpoint if the current user has edit permission on the org)",
             readonly=True,
         ),
+        "last_login_at": fields.Raw(
+            attribute=lambda o: o.last_login_at if check_can_access_user_private_info() else None,
+            description="The user last connection date (only present on show organization endpoint if the current user has edit permission on the org)",
```

Comment:
I think all members have access (same for email)
```suggestion
            description="The user last connection date (only present on show organization endpoint if the current user is member of the organization)",
```

Reply:
The check is on `OrganizationAdminNeed(org.id), OrganizationEditorNeed(org.id)`, is it the edit permission? Or any member is `Editor`?

Reply:
Any member is `Editor` indeed (poorly named indeed)!
There is a dedicated `EditOrganizationPermission` with `OrganizationAdminNeed` only need.

Reply:
Do we want to change it to allow admin only access?

Reply:
I think going with `Editor` is a good idea for now!

Reply:
I think we need to reword the description if we want to go with all members (i.e. `Editor`) :)